react/features/chat/components/web/EmojiSelector.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const EmojiSelector = ({ onSelect }) => {
+    const emojiMap = {
```

Comment:
Until we make these options configurable, let's put here the commonly used reactions: 👍, ❤️,  😂, 😮 / 🔥, 😢, 🙏  or similar.